src/libNode/DSBlockProcessing.cpp

```diff
@@ -536,6 +529,10 @@ bool Node::ProcessDSBlock(const vector<unsigned char>& message,
     // Add to block chain and Store the DS block to disk.
     StoreDSBlockToDisk(dsblock);
 
+    map<PubKey, Peer> dsPoWWinners = m_mediator.m_dsBlockChain.GetLastBlock()
```

Comment:
Since dsPoWWinners only be used inside if (!LOOKUP_NODE_MODE), it's better to move backward (around line 558)